### ISSUE TITLE
setup_default_capsule should handle situations when a FQDN is not present

### ISSUE THREAD

**Issue by omaciel** [SatelliteQE/automation-tools#105]
If you don't configure a server with a valid ``fqdn``, then ``setup_default_capsule`` will fail with the following stack trace:

```bash
Executing task 'setup_default_capsule'
Traceback (most recent call last):
  File "/home/jenkins/shiningpanda/jobs/90156090/virtualenvs/d41d8cd9/lib/python2.7/site-packages/fabric/main.py", line 743, in main
    *args, **kwargs
  File "/home/jenkins/shiningpanda/jobs/90156090/virtualenvs/d41d8cd9/lib/python2.7/site-packages/fabric/tasks.py", line 384, in execute
    multiprocessing
  File "/home/jenkins/shiningpanda/jobs/90156090/virtualenvs/d41d8cd9/lib/python2.7/site-packages/fabric/tasks.py", line 274, in _execute
    return task.run(*args, **kwargs)
  File "/home/jenkins/shiningpanda/jobs/90156090/virtualenvs/d41d8cd9/lib/python2.7/site-packages/fabric/tasks.py", line 174, in run
    return self.wrapped(*args, **kwargs)
  File "/home/jenkins/workspace/satellite6-installer/automation_tools/__init__.py", line 968, in product_install
    execute(setup_default_capsule, host=host)
  File "/home/jenkins/shiningpanda/jobs/90156090/virtualenvs/d41d8cd9/lib/python2.7/site-packages/fabric/tasks.py", line 384, in execute
    multiprocessing
  File "/home/jenkins/shiningpanda/jobs/90156090/virtualenvs/d41d8cd9/lib/python2.7/site-packages/fabric/tasks.py", line 274, in _execute
    return task.run(*args, **kwargs)
  File "/home/jenkins/shiningpanda/jobs/90156090/virtualenvs/d41d8cd9/lib/python2.7/site-packages/fabric/tasks.py", line 174, in run
    return self.wrapped(*args, **kwargs)
  File "/home/jenkins/workspace/satellite6-installer/automation_tools/__init__.py", line 302, in setup_default_capsule
    domain = hostname.split('.', 1)[1]
IndexError: list index out of range
```

**Comment by elyezer** [comment]
@omaciel what you suggest to do when this happens? Abort the installation or try some workaround?

**Comment by omaciel** [comment]
@elyezer maybe default to `localhost.localdomain`?

**Comment by elyezer** [comment]
One of the requisites of Satellite is "Full forward and reverse DNS resolution using a fully qualified domain name". With that said I think we can fail the task if it is not available.

**Comment by elyezer** [comment]
@omaciel any thoughts based on the last comment?

**Comment by rplevka** [comment]
closing this per merged PR #294
